spec/features/ministers_spec.rb

```diff
@@ -27,10 +37,10 @@
 
 private
 
-  def ministers_content_hash
-    @content_hash = {
-      title: I18n.t("ministers.title"),
-      details: {},
-    }
+  def load_content_item(file_name)
```

Comment:
Interesting, I didn't know about that!

I'm testing a few specific edge cases, that were discovered thanks to the particular contents of the content item at this time. If we keep the examples in a separate app, isn't there a risk of these specs starting to fail if someone changes the examples in the future?

Reply:
The examples in Publishing API are also used to validate the schema files (when the content schemas are regenerated), so including the edge cases there is quite possibly the right place for them, since we'll ensure they remain valid against the schema.